### ISSUE TITLE
SUPPORT : what is your conclusion about Ionic + LWC integration ?

### ISSUE THREAD

**Issue by RupertBarrow** [pmdartus/ionic-lwc-integration#1]
Hi PM,

After this test, what is the conclusion about integrating LWC (OSS) components into an Ionic app : 
- is the result worth the effort ?
- is it possible to integrate LWC views, or should we stick to *.tsx ?
- is this still a valid idea today, 2 years later, after evolution of Ionic and LWC/LWR ?
- what should we expect from LWR (Node) ?  will it generate static code we can embed into an Ionic app ?  Or will the LWR runtime support features (including native, routing, service workers, etc.) currently offered by Ionic ?

In a nutshell, what solution do you suggest to build a serious mobile LWC (OSS) app ?
